Modules/Loadable/Markups/VTKWidgets/vtkSlicerMarkupsWidgetRepresentation2D.cxx

```diff
@@ -200,6 +307,10 @@ void vtkSlicerMarkupsWidgetRepresentation2D::UpdateAllPointsAndLabelsFromMRML(do
     activeControlPointIndex = activeControlPointIndices[0];
     }
 
+  vtkNew<vtkPoints> pts;
```

Comment:
These variables should not be recreated at each call of the `UpdateAllPointsAndLabelsFromMRML` method (it would cause memory fragmentation and slowdown if you reallocate this array dozens of times per second, as you move a markup). Instead, the points and cells objects should be created once, where `LabelsLinePolyData` is created.